src/user-guide/mode.md

```diff
@@ -29,6 +29,10 @@ Forecasts have several additional metadata attributes (available via the `/metad
 * `max_forecast_offset` is the maximum allowed value for the `forecast_offset` parameter.
 * `forecast_length` describes how many time steps a single forecast has.
 
+```{note}
+Due to their large size only the last few forecasts are available. This means, that specifying `start` and `end` outside of `last_forecast_reftime`there will be no result.
```

Comment:
Specifying `start` and `end` outside of `last_forecast_reftime` will yield no result.